components/email/src/Email.svelte

```diff
@@ -650,22 +665,42 @@
 
   $: {
     if (activeThread) {
-      attachedFiles = activeThread.messages.reduce((files, message) => {
-        for (const [fileIndex, file] of message.files.entries()) {
-          if (file.content_disposition === "attachment") {
-            if (!files[message.id]) {
-              files[message.id] = [];
+      attachedFiles = activeThread.messages.reduce(
```

Comment:
CHeck if there is an already existing reactive binding for this and use that instead